pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -356,7 +356,7 @@
         "4.3.0",
         "4.3.1"
       ],
-      "BaselineVersion": "4.3.0"
```

Comment:
You may have only needed to do this for the RID-agnostic ones, but it doesn't hurt to do for all.